sdk/purview/purview-workflow-rest/test/public/userrequest.spec.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// import { PurviewWorkflowClient } from "@azure-rest/purview-workflow";
+import { Recorder } from "@azure-tools/test-recorder";
+import { createClient } from "./utils/recordedClient";
+
+import { Context } from "mocha";
+import { PurviewWorkflowClient } from "../../src/clientDefinitions";
+import { SubmitUserRequestsParameters } from "../../src/parameters";
+import { isUnexpected } from "../../src/isUnexpected";
+
+describe("Submit a user request", () => {
+  let recorder: Recorder;
+  let client: PurviewWorkflowClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = new Recorder(this.currentTest);
+    client = await createClient(recorder);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("should submit a user request.", async function () {
+    const options: SubmitUserRequestsParameters = {
+      body: {
+        comment: "Thanks!",
+        operations: [
+          {
+            type: "CreateTerm",
+            payload: {
+              glossaryTerm: {
+                name: "term",
+                anchor: { glossaryGuid: "5dae5e5b-5aa6-48f1-9e46-26fe7328de71" },
+                nickName: "term",
+                status: "Approved",
+              },
+            },
+          },
+        ],
+      },
+    };
+    const result = await client.path("/userrequests").post(options);
+
+    if (isUnexpected(result)) {
+      throw result.body.error;
+    }
+  });
```

Comment:
You should assert that the result you got cointains the information you expect. We can't just rely on not throwing.

Reply:
Fixed, thanks, but for workflowtasks.spec.ts, there have no result in response, it just return status code, so I didn't verify result in this test.